packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue

```diff
@@ -1,28 +1,51 @@
 <template>
-  <SmartLink :item="{ link: '/docs/reference/releases-at-okta/' }">
-    <span class="api-label api-label-beta" v-if="access == 'beta'">
+  <SmartLink :item="{ link }">
+    <span class="api-label api-label-beta" v-if="access === labelType.BETA">
       <i class="fa fa-warning"></i> Beta
     </span>
-    <span class="api-label api-label-ea" v-if="access == 'ea'">
+    <span class="api-label api-label-ea" v-if="access === labelType.EARLY_ACCESS">
       <i class="fa fa-flag"></i> Early Access
     </span>
-    <span class="api-label api-label-deprecated" v-if="access == 'deprecated'">
+    <span class="api-label api-label-deprecated" v-if="access === labelType.DEPRECATED">
       <i class="fa fa-fire-extinguisher"></i> Deprecated
     </span>
+    <span class="api-label api-label-ie" v-if="access === labelType.IDENTITY_ENGINE">
+      Identity Engine
```

Comment:
```suggestion
      Identity engine
```
this is the agreed upon capitalization when referring to the Identity engine. FYI